p2p/discovery.go

```diff
@@ -42,7 +40,7 @@ type discovery struct {
 	mdns libp2pdiscovery.Service
 }
 
-// newDiscovery creates a new discovery instance from the host
+// newDiscovery creates a new discovery submodule
```

Comment:
idk if the right word is submodule, I think instance is more correct?

Reply:
will fix

Reply:
fixed